cogs/misc.py

```diff
@@ -420,6 +429,15 @@ async def parse(self, ctx, *, time_and_arg: converters.HumanTime(other=True)):
                            f'Other argument: {time_and_arg.other}\n'
                            f'Delta: {(time - pendulum.instance(ctx.message.created_at)).in_words()}')
 
+    @commands.command()
+    async def someone(self, ctx, *, text)
+        """
+        Repeats the message but with a random member's name after :someone
+        """
+        emote = choice(self.someone_emotes)
+        user = choice(ctx.guild.members).display_name
```

Comment:
This will fail because `choice` is not defined. Module `random` is imported at the top of file so use `random.choice`

Reply:
Whoops, missed that when copying from my project